### DISCUSSION TITLE
Bamboo stopped building my project a few days ago

### DISCUSSION THREAD

**Discussion by MatthiasMD80** [green-code-initiative/creedengo-php#37] [Q&A]
Hello,

when Bamboo builds my project, I get the following error message:
Initialization of bean failed; nested exception is java.lang.IllegalStateException: The plugin [ecocodephp] does not support Java 11.0.13: fr/greencodeinitiative/php/PHPPlugin has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0 -> [Help 1]

What could be the problem?

Thank you for help.

**Comment by dedece35** [comment]
Hi @MatthiasMD80,
the problem is quite clear ... the PHP plugin used is compiled with a JDK 17 and you try to use and JDK 11 ... try with a JDK 17 and check.
